tests/hypervisor_test.c

```diff
@@ -524,12 +524,12 @@ START_TEST(test_cc_oci_vm_args_get) {
 	g_strfreev (args);
 
 	/* recreate the args file */
-	ret = g_file_set_contents (args_file, "hello\# comment\n", -1, NULL);
+	ret = g_file_set_contents (args_file, "hello\\# comment\n", -1, NULL);
```

Comment:
Good catch - but then this hints to me that the test was not really working or maybe testing what it should... @devimc can you have a think about that?